src/Windowing/Silk.NET.Windowing.Common/Interfaces/IMonitor.cs

```diff
@@ -1,17 +1,46 @@
-// This file is part of Silk.NET.
-// 
-// You may modify and distribute Silk.NET under the terms
-// of the MIT license. See the LICENSE file for details.
-
-using System.Drawing;
-
-namespace Silk.NET.Windowing.Common
-{
-    public interface IMonitor : IWindowHost
-    {
-        string Name { get; }
-        int Index { get; }
-        Rectangle Bounds { get; }
-        VideoMode VideoMode { get; }
-    }
-}
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System.Drawing;
+
+namespace Silk.NET.Windowing.Common
+{
+    /// <summary>
+    /// An interface representing a monitor.
+    /// </summary>
+    public interface IMonitor : IWindowHost
+    {
+        /// <summary>
+        /// The name of this monitor.
+        /// </summary>
+        string Name { get; }
+        
+        /// <summary>
+        /// The index of this monitor.
+        /// </summary>
+        int Index { get; }
+        
+        /// <summary>
+        /// The bounds of this monitor.
+        /// </summary>
+        Rectangle Bounds { get; }
+        
+        /// <summary>
+        /// The current video mode of this monitor.
+        /// </summary>
+        VideoMode VideoMode { get; }
+        
+        /// <summary>
+        /// This monitor's gamma correction.
+        /// </summary>
+        float Gamma { get; set; }
+
+        /// <summary>
+        /// Get all video modes that this monitor supports.
+        /// </summary>
+        /// <returns>An array of all video modes.</returns>
+        VideoMode[] GetAllVideoModes();

```

Comment:
Is there a specific reason you opted for an array and not `IEnumerable<VideoMode>` here? Later on you seem to do an additional copy from list to array so I think making it IEnumerable may be more efficient.

Reply:
Agree with @Perksey. I also would remove the mentioned `ToArray` call.